constants.js

```diff
@@ -78,6 +78,12 @@ const BLOCKCHAINS = {
 
 const INCENTIVE_TYPE = {
     NEUROWEB: 'Neuroweb',
+    ERC20: 'NeurowebERC20',
```

Comment:
Does it make sense to call it `NEUROWEB_ERC20`? It can't be any ERC20 token just yet